### PR TITLE
DQA-6074: Create .gitattributes file for toolkit.

### PR DESCRIPTION
**Pull request by zarabatana**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+*.*ignore         text
+
+# Text files where line endings should be preserved
+*.patch    -text
+
+#
+# Exclude files from exporting
+#
+
+.gitattributes     export-ignore
```
Comment (joaocsilva) [inline .gitattributes:173]: I think we can exclude some more files, what do you think?
```suggestion
.drone.yml         export-ignore
.editorconfig      export-ignore
.eslintrc.json     export-ignore
.gitattributes     export-ignore
```

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+.gitignore         export-ignore
+.gitkeep           export-ignore
+.github/           export-ignore
+tests/             export-ignore
+docker-compose.yml export-ignore
+phpcs.xml          export-ignore
+phpdoc.dist.xml    export-ignore
+phpmd.xml          export-ignore
+phpunit.xml.dist.  export-ignore
```
Comment (joaocsilva) [inline .gitattributes]: Typo
```suggestion
phpunit.xml.dist   export-ignore
```

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+# *.svg           binary
+*.svgz            binary
+*.tif             binary
+*.tiff            binary
+*.wbmp            binary
+*.webp            binary
+
+# Scripts
+*.fish     text eol=lf
+*.zsh      text eol=lf
```
Comment (joaocsilva) [inline .gitattributes]:
```suggestion
*.fish            text eol=lf
*.zsh             text eol=lf
```

File: .gitattributes [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+*.pyc             binary
+
+# RC files (like .babelrc or .eslintrc)
+*.*rc             text
+
+# Ignore files (like .npmignore or .gitignore)
+*.*ignore         text
+
+# Text files where line endings should be preserved
+*.patch    -text
```
Comment (joaocsilva) [inline .gitattributes]:
```suggestion
*.patch           -text
```

**Review by joaocsilva — CHANGES_REQUESTED** [review]
Hi @zarabatana , 
I'm suggesting some more files to be added and fixing a typo in the phpunit.xml.dist file.
The rest is just alignment

**Review by jonhy81 — APPROVED** [review]

**Review by joaocsilva — APPROVED** [review]

**Pull request merged**